packages/form/addon/components/cf-field/input/radio.hbs

```diff
@@ -23,7 +23,7 @@
     {{/if}}
   </label>
 {{/each}}
-{{#if (and @field.optional @field.answer.value)}}
+{{#if (and (and @field.optional @field.answer.value) (not @disabled))}}
```

Comment:
```hbs
{{#if (and @field.optional @field.answer.value (not @disabled)}}
```

No need for the additional `and`

Reply:
i was searching in the documentation if `and` accept more then two parameter, but i did not see it, all examples ware explaining with two options only, but it was in my mind, i will do it in new PR

Reply:
Yeah I was very convinced that I'd previously used it with more than 2 parameters but had to check the code to be 100% sure: https://github.com/jmurphyau/ember-truth-helpers/blob/master/packages/ember-truth-helpers/src/helpers/and.ts

The helper loops over **all** parameters and returns `false` if any one of them is falsey.

Reply:
Yeah I was very convinced that I'd previously used it with more than 2 parameters but had to check the code to be 100% sure: https://github.com/jmurphyau/ember-truth-helpers/blob/master/packages/ember-truth-helpers/src/helpers/and.ts

The helper loops over **all** parameters and returns `false` if any one of them is falsey.